index.html

```diff
@@ -0,0 +1,225 @@
+            <p>
+                Use Nova theme for your next web project. It is completely
+                customizable so you can change any of the sections to fit your
+                needs. Nova Theme is Free for any kind of use, personal and
+                commercial. Have fun and good luck!
+            </p>
+       </article>
+    </section>
+
+
+    <section class="pricing row" id="Pricing">
+
+        <div class="pricing__container container info">
+            <h2>CHOOSE YOUR PRICING PLAN</h2>
+            <h4>Pick any of our super affordable pricing plans</h4>
+        </div>
+
+        <div class="pricing__container container">
+
+            <article class="pricing__article cell info">
+                <h3 class="pricing__option">Basic</h3>
+                <h1>$15</h1>
+                <h5>per month</h5>
+                <ul>
+                    <li class="strong">Up to 7 Projects</li>
+                    <li class="strong">2 Additions Developers</li>
+                    <li class="strong hidden">2 Additions Developers</li>
+                    <li class="strong hidden">2 Additions Developers</li>
+
+                </ul>
+                <button class="pricing__button"><a>Get Started</a></button>
+            </article>
+
+            <article class="pricing__article cell info">
+                <h3 class="pricing__option">AGENCY</h3>
+                <h1>$55</h1>
+                <h5>per month</h5>
+                <ul>
+                    <li class="strong">Up to 25 Projects</li>
+                    <li class="strong">2 additions Developers</li>
+                    <li class="strong">Unlimited Support</li>
+                    <li class="strong hidden">Unlimited Support</li>
+                </ul>
+                <button class="pricing__button"><a>Get Started</a></button>
+            </article>
+               
+            <article class="pricing__article cell info">
+                <h3 class="pricing__option">PRO</h3>
+                <h1>75$</h1>
+                <h5>per month</h5>
+                <ul>
+                    <li class="strong">Up to 25 Projects</li>
+                    <li class="strong">2 additions Developers</li>
+                    <li class="strong">Unlimited Support</li>
+                    <li class="strong">1.5GB Disk Space</li>
+                </ul>
+                <button class="pricing__button"><a>Get Started</a></button>
+            </article>
+
+        </div>
+
+    </section>
+
+
+    <section class="team row" id="Team">
+        
+        <div class="team__container container info">
+            <h2>THE TEAM</h2>
+        </div>
+
+        <div class="team__container container">
+
+            <article class="team__article cell info">
+                <img src="./images/cto.png" alt="">
+                <h3>CTO</h3>
+                <p class="strong">Johhny B Good</p> 
+                <p>The brains behind whole operation</p>
+                <div class="media__container">
```

Comment:
Skoro mamy Element z BEM (`media__container`) to gdzieś wyżej w hierarchii powinien być Block tj. `media` - tego nie widzę.